common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -434,6 +435,7 @@ def setUp(self):
         AutoAuthPage(self.browser, username=self.USERNAME, email=self.EMAIL,
                      course_id=self.course_id, staff=False).visit()
 
+    @skip('Disable temporarily to get course bookmarks out')
```

Comment:
@robrap I cannot work out why this test is failing in this branch, so I'm disabling it so we can merge the story and land the feature. It looks like you changed this method so I'd love your insight as to what it is intended to do.